### ISSUE TITLE
Re-layout protocol module functions

### ISSUE THREAD

**Issue by GLolol** [GLolol/PyLink#156]
The current structure of protocol modules is, quite frankly, a brutal mess. 

1) Why have two different commands for sending things like kick (`irc.proto.kickServer` and `irc.proto.kickClient`) when they can just be merged into `irc.proto.kick` and still have the sender field work with both UIDs and SIDs?
- Many protocols already have that with a `_sendKick` function which `kickServer` and `kickClient` wrap around.
- [x] **Done**: acdd7dbb782765f581...2fd0a8ae741a663d

2) InspIRCd and TS6 protocol modules are still doing authentication in the `handle_events` block. Ew. 
- The [newer approach](https://github.com/GLolol/PyLink/blob/ff6841c/protocols/unreal.py#L504-L531), which I used to write the `unreal` module, is a lot more elegant by coersing messages without a sender prefix to the uplink SID, and sending them to handler functions properly.
- [x] TS6: 814ebc9fe0f897318a3ecd89ca6a90e65eecc963
- [x] InspIRCd bb9d87b

tl;dr When I first started writing PyLink, all this module structuring was new to me, and I don't like how I laid out the protocol modules then. Most of these will probably be breaking changes, so it's best to get this done before I ever release v1.0 (and drop the old names entirely).

**Comment by GLolol** [comment]
3) Break `updateClient` into separate functions for `CHGHOST`/`CHGIDENT`/`CHGNAME`?
